src/app/components/Button/Button.module.css

```diff
@@ -15,3 +17,15 @@
 .button--secondary {
   background: var(--secondary-color);
 }
+
+.button--primary:focus {
+  background: var(--primary-clicked-color);
+  transform: scale(1.025);
+  box-shadow: 0 0 0.7em 0 var(--primary-clicked-color);
```

Comment:
I am not sure what `clicked` stands for in `--primary-clicked-color`.
`clicked` is not a good name here, because it's an event which triggers something (what is triggered?). a button could be tapped, clicked or pressed (like pressing space on your keyboard).

Reply:
Thanks for the review. I have also noticed the bug in the input field. I tried to solve it by using `(.container__input:not(:placeholder-shown).container__input:not(:focus)` to check whether something has been written to the input field, and the input field is not in focus. Unfortunately without success